### PR TITLE
Overlay on activate

### PR DESCRIPTION
**Pull request by murugaratham**
for issue #105, for your review

### CODE REVIEW

File: src/Inspector/Overlay.tsx [hunk 1]
```diff
@@ -143,7 +143,7 @@ class OverlayTip {
       color: '#d7d7d7',
     })
 
-    this.tip.style.zIndex = '10000000'
+    this.tip.style.zIndex = '10000001'
```
Comment (zthxxx) [inline src/Inspector/Overlay.tsx:146]: is this really necessary?

this hardcode mirror from https://github.com/facebook/react/blob/v16.13.1/packages/react-devtools-shared/src/backend/views/Highlighter/Overlay.js#L44

File: src/Inspector/Inspector.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect, useRef, ReactElement } from 'react'
 import type { Fiber } from 'react-reconciler'
 import hotkeys, { KeyHandler } from 'hotkeys-js'
-import { setupHighlighter } from './utils/hightlight'
+import { setupHighlighter } from './utils/highlight'
```
Comment (zthxxx) [inline src/Inspector/Inspector.tsx:4]: thanks for typo 😂

File: src/Inspector/Overlay.tsx (hunk 1 above)
Comment (murugaratham) [reply src/Inspector/Overlay.tsx:146]: I didn't want to adjust this too, but without this, when keys are pressed, the overlay is behind some blue overlay, you can try cloning from my branch

File: src/Inspector/Inspector.tsx (hunk 2 above)
Comment (murugaratham) [reply src/Inspector/Inspector.tsx:4]: 不客气😊

File: src/Inspector/Inspector.tsx [hunk 3]
```diff
@@ -103,6 +103,11 @@ export const Inspector: React.FC<InspectorProps> = (props) => {
     overlay.setRemoveCallback(stopCallback)
 
     overlayRef.current = overlay
+
+    const { scrollLeft, scrollTop } = document.documentElement
+    const element = document.elementFromPoint(scrollLeft, scrollTop)
+
+    handleHoverElement(element as HTMLElement)
```
Comment (zthxxx) [inline src/Inspector/Inspector.tsx:110]: I try it online, but feels the interaction is weird.

Maybe I want is inspect at mouse point at start, so we need listen mouse move and recode point at this Inspector component mounted.

Comment (murugaratham) [reply src/Inspector/Inspector.tsx:110]: My opinion is that, when hot keys are pressed, if user doesn't move mouse, the overlay doesn't trigger, there's no visual indicator as well. Having the overlay appearing upon hotkeys goes the developers a sense that there's something happening. My use-case is to create a cra template and pre install this tool, but newer developers might get a bit lost

File: src/Inspector/Overlay.tsx (hunk 1 above)
Comment (zthxxx) [reply src/Inspector/Overlay.tsx:146]: please list some screenshots or a online page?

Comment (murugaratham) [reply src/Inspector/Overlay.tsx:146]: Screenshot 2021-11-02 at 2 51 40 PM

**Comment by zthxxx** [comment]
release in react-dev-inspector@1.7.1

**Pull request closed without merging**